Package.swift

```diff
@@ -62,7 +62,7 @@ let package = Package(
         .target(
             name: "BraintreeApplePay",
             dependencies: ["BraintreeCore"],
-            publicHeadersPath: "Public"
+            path: "Sources/BraintreeApplePay"
```

Comment:
Also don't think we need to specify a path here since we're following the SPM default package structure `Sources/<TARGET_NAME>`

[Apple docs on path](https://developer.apple.com/documentation/packagedescription/target/path)

Reply:
Yeah it looks like some of our Swift packages contain and `path` (SEPA and NXO) and some do not. We can remove it here and should have a follow up to remove from SEPA and NXO as well!